showcase/gapic-showcase/pom.xml

```diff
@@ -132,14 +132,35 @@
 
     <!-- Test dependencies -->
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
+      <groupId>org.junit.platform</groupId>
```

Comment:
We probably don't need this `junit-platform-launcher`.

Reply:
Removed